.github/workflows/golangci-lint.yml

```diff
@@ -14,14 +14,28 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
+      - name: Generate Golang
+        run: |
+          export PATH=$PATH:/home/runner/go/bin/
+          make embed_files
+
+      - id: configfile
+        run: |
+          if [ "${{ github.event_name }}" == "pull_request" ]; then
+            echo "::set-output name=file::.golangci.yml"
+          else
+            echo "::set-output name=file::.golangci-schedule.yml"
+          fi
+
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.39
+          version: v1.42.1
```

Comment:
Does this specify the patch version (trailing `.1`) ? Contrary to the comment ?

Reply:
It does :(